src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,75 @@
+@import "utils/variables";
+
+.clock {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 100vh;
+
+  &__dial {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    width: $width-clock-dial;
+    height: $width-clock-dial;
+
+    border-radius: 50%;
+    background-color: $color-clock-dial;
+  }
+
+  &__second-hand {
+    position: relative;
+    top: $top-shift-clock-second-hand;
+
+    width: 0;
+    height: $height-clock-second-hand;
+
+    border-right: $width-clock-second-hand solid $color-clock-second-hand;
+    border-radius: $border-radius-clock-hand;
+
+    transform-origin: bottom;
+    animation-name: rotate;
+    animation-duration: $animation-duration-clock-second-hand;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+  }
+
+  &__circle {
+    position: relative;
+    top: $top-shift-clock-circle;
+    z-index: 1;
+
+    width: $width-clock-circle;
+    height: $width-clock-circle;
+
+    border-radius: 50%;
+    background-color: $color-clock-circle;
+  }
+
+  &__minute-hand {
+    position: relative;
+    top: $top-shift-clock-minute-hand;
+
+    border-right: $width-clock-minute-hand solid $color-clock-minute-hand;
+    border-radius: $border-radius-clock-hand;
+    width: 0;
+    height: $height-clock-minute-hand;
+
+    transform-origin: bottom;
+    animation-name: rotate;
+    animation-duration: $animation-duration-clock-minute-hand;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+  }
```

Comment:
you have a repeating in second and minute hands. avoid it by using extend or common class